### ISSUE TITLE
Inconsistent tone when selecting "Stay true to my color inputs"

### ISSUE THREAD

**Issue by ztwhaley** [material-foundation/material-color-utilities#160]
When selecting the option to stay true to color input, the Primary color does not come out as P-40 in the color system anymore, nor is it the same as the input color. The color I tested (#0646D1), which has a tone of 36, resulted in an output Primary color with a tone of 27. The Secondary color maintained a tone of 40 as advertised, and the Tertiary color seemed to match the tone chosen for the Primary color at 27. Error color is maintained at 40.

I thought one of the main advantages of using this system was to be able to maintain similar tone levels across colors to make other things easier such as ensuring accessibility. The tone consistency between primary, secondary, tertiary, and error is maintained when "Stay true to my color inputs" is disabled.

Why this inconsistency in tone when this is option enabled, and why the seemingly arbitrary tone shift from the input color (i.e., going from 36 to 27)?

**Comment by WebKun** [comment]
You mean you thought turn on "stay true to color input" this option, the Primary color's hue chroma as will same with input color, but tone is still at 40?

**Comment by sailr7** [comment]
The hue stayed the same. But the chroma and tone changed even when I selected "stay true to color input". However, the output Primary color still says "P-40" (meaning Tone of 40; see attached), which is incorrect. To be precise, the input color HCT was H:275, C:69, T:36; and the output color HCT was H:275, C:60, T:27.

My confusion can be broken down into three related questions:
1. Why did it change the Primary color even though I selected "stay true to color input"?
2. Regarding the change it made to the Primary color, why did it decide to reduce Chroma by 9, and Tone by 9?
3. Why does it advertise as P-40 for the Primary, Secondary, and Tertiary colors, when in reality the Tone of the Primary (and Tertiary) colors are not really 40?

![Screenshot 2024-07-31 222818](https://github.com/user-attachments/assets/49cc7b49-dfbe-42bd-8de4-7ddcf6cc07dc)

**Comment by WebKun** [comment]
About always show "P-40" in primary, it might be they forgot to let this label update when color changing.

**Comment by WebKun** [comment]
for the question 1&2, maybe this is answer
https://m3.material.io/styles/color/advanced/adjust-existing-colors#cb49eeb4-3bbd-4521-9612-0856c27f91ef

**Comment by sailr7** [comment]
> for the question 1&2, maybe this is answer https://m3.material.io/styles/color/advanced/adjust-existing-colors#cb49eeb4-3bbd-4521-9612-0856c27f91ef

Yes, I looked at this, but based on what it says, if you toggle the “match color” option on, then it should disable fidelity and match the color. The issue I described above was with this option toggled on, and it is still occurring. I think you may be right that they forgot to update the label when color changing, however I am curious how the formula works if they are making a change and why they are making the change.

**Comment by Harm-Nullix** [comment]
Isn't this more of a material-theme-builder question then a material-color-util question?

**Comment by adamshields** [comment]
> The hue stayed the same. But the chroma and tone changed even when I selected "stay true to color input". However, the output Primary color still says "P-40" (meaning Tone of 40; see attached), which is incorrect. To be precise, the input color HCT was H:275, C:69, T:36; and the output color HCT was H:275, C:60, T:27.
> 
> My confusion can be broken down into three related questions:
> 
> 1. Why did it change the Primary color even though I selected "stay true to color input"?
> 2. Regarding the change it made to the Primary color, why did it decide to reduce Chroma by 9, and Tone by 9?
> 3. Why does it advertise as P-40 for the Primary, Secondary, and Tertiary colors, when in reality the Tone of the Primary (and Tertiary) colors are not really 40?
> 
> ![Screenshot 2024-07-31 222818](https://private-user-images.githubusercontent.com/32799575/354054924-49cc7b49-dfbe-42bd-8de4-7ddcf6cc07dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwNTkxODgsIm5iZiI6MTczMDA1ODg4OCwicGF0aCI6Ii8zMjc5OTU3NS8zNTQwNTQ5MjQtNDljYzdiNDktZGZiZS00MmJkLThkZTQtN2RkY2Y2Y2MwN2RjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI3VDE5NTQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmMzZjNDQzZDdiZGMxN2RjNTBkNDZhMzI0N2NiNDE4ZWUxNjNhMTUzNTFkMzNiZmYzZmM2MDBhYjhiYTJlOWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZiQzG9BOkHghGZD0eWl-fF2ACxYi4EMgn3eTdamjxyQ)

I am running into the same issue. @Harm-Nullix this package is used winin the material-theme-builder if i am not mistaken. Thats how i ended up here
